src/styles/main.scss

```diff
@@ -0,0 +1,89 @@
+// Color variables
+$border-color: #000;
+$day-color: #eee;
+$hover-color: #ffbfcb;
+
+// Size variables
+$cell-size: 100px;
+$padding: 10px;
+$gap: 1px;
+$days-in-a-week: 7;
+$days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+// Page styles
+.page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  margin: 0;
+  height: 100vh;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+// Calendar styles
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  gap: $gap;
+
+  padding: $padding;
+
+  width: $days-in-a-week * ($cell-size + $gap);
+
+  &__day {
+    display: flex;
+    position: relative;
+
+    height: $cell-size;
+    width: $cell-size;
+
+    box-sizing: border-box;
+    border: 1px solid $border-color;
+
+    background-color: $day-color;
+
+    transition:
+      transform 0.5s,
+      background-color 0.5s;
+
+    @for $i from 1 through 31 {
+      &:nth-of-type(#{$i})::before {
+        content: '#{$i}';
+        position: absolute;
+        bottom: ($cell-size - 2) / 2;
+        right: ($cell-size - 3) / 2;
+        transform: translate(50%, 50%);
```

Comment:
- These styles are the same for all of them, so you can move outside the loop.
- It's better to avoid position absolute, to center the element you can use display flex

Reply:
I tried doing it with: 
` display: flex;
  align-items: center;
  justify-content: center;` in calendar__day but when i do so, there's a pixel problem because of which i cant pass the test. So i once again used position absolute but moved it out from loop.
![image](https://github.com/user-attachments/assets/8d0dfe1a-9b63-4310-a377-4e4c0b883b7e)